scripts/c2r_script.py

```diff
@@ -8,6 +9,11 @@
 
 from urllib2 import urlopen, URLError
 
+# NOTE: What are all possible levels we can get from rhc?
```

Comment:
How about trace? :thinking:

Reply:
We should stick with the levels from python, actually... I think what we can do is try to parse the level, if it is not a level that exists in the python mapping, we then default it to info.